pkg/kv/postgres/store.go

```diff
@@ -41,6 +44,7 @@ const (
 	DefaultMaxOpenConnections    = 25
 	DefaultMaxIdleConnections    = 25
 	DefaultConnectionMaxLifetime = 5 * time.Minute
+	DefaultScanPageSize          = 300
```

Comment:
Actually (assuming you tested using a local instance of postgres with 0 network latency), I believe it was tested before with the similar db iterator and 1000 was the optimal value.

Reply:
So, `1000` should be the default?
I'm not really opinionated about it, as my test show really similar results and as you said - it was local and not necessarily representing a production system

Reply:
Done - back to 1000